cryptus-app/components/wallet_viewer/nft_details/nft_details.module.scss

```diff
@@ -0,0 +1,12 @@
+@use "/styles/variables" as v;
+
+.container {
+    flex: 1 1 auto;
+    margin: 16px;
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    max-width: 420px;
```

Comment:
use global variable v.$mobile-width (and it is 375 I think, not 420px), and use width not max-width

Reply:
j'ai gardé max-width pour être responsive